saleor/graphql/translations/types.py

```diff
@@ -131,6 +107,39 @@ def resolve_attribute(root: attribute_models.Attribute, _info):
         return root
 
 
+class AttributeValueTranslatableContent(ModelObjectType):
+    id = graphene.GlobalID(required=True)
+    name = graphene.String(required=True)
+    rich_text = JSONString(description="Attribute value." + RICH_CONTENT)
+    plain_text = graphene.String(description="Attribute plain text value.")
+    translation = TranslationField(
+        AttributeValueTranslation, type_name="attribute value"
+    )
+    attribute_value = graphene.Field(
+        "saleor.graphql.attribute.types.AttributeValue",
+        description="Represents a value of an attribute.",
+        deprecation_reason=(
+            f"{DEPRECATED_IN_3X_FIELD} Get model fields from the root level queries."
+        ),
+    )
+    attribute = graphene.Field(
+        AttributeTranslatableContent,
+        description="Associated attribute that can be translated." + ADDED_IN_38,
+    )
+
+    class Meta:
+        model = attribute_models.AttributeValue
+        interfaces = [graphene.relay.Node]
+
+    @staticmethod
+    def resolve_attribute_value(root: attribute_models.AttributeValue, _info):
+        return root
+
+    @staticmethod
+    def resolve_attribute(root: attribute_models.AttributeValue, info):
+        return AttributesByAttributeValueIdLoader(info.context).load(root.attribute_id)
```

Comment:
Is the name of the data loader correct? It loads data using attribute ID, not attribute value ID.